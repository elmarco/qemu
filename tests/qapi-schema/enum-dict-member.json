{ 'enum': 'TestEnum', 'data': [ { 'name': 'foo' }, { 'name': 'bar' } ]}
