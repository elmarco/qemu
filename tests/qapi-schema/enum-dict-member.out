enum QType ['none', 'qnull', 'qnum', 'qstring', 'qdict', 'qlist', 'qbool']
    prefix QTYPE
enum TestEnum ['foo', 'bar']
object q_empty
